clientconn_test.go

```diff
@@ -34,6 +34,7 @@
 package grpc
 
 import (
+	"context"
```

Comment:
This is for go1.7. This is the reason you saw the errors on travis. Please use "golang.org/x/net/context".